src/Netflix/Netflix.ts

```diff
@@ -333,7 +373,7 @@ export default <IPluginBase & IPlugin>{
         normal: false
       },
       {
-        name: "ChangeText",
+        name: "Change Text",
```

Comment:
Didn't catch this last time, but "Change Text" could be clearer. Perhaps "Change Subtitle Language" Also, the match phrase could have an "[/language]" after "[text/subtitle]"

Reply:
Changing "Change Text" to "Change Subtitle Language". FYI `Text` is Netflix's formal definition of subtitle.

Skipping the "[/language]". The command "Change Audio" also deals with language. That makes the word "language" ambiguous to the user.